index.bs

```diff
@@ -1797,6 +1822,48 @@ a [=source type=] |sourceType|, and a [=duration=] |expiry|:
     1. Set |startDuration| to |endDuration|.
 1. Return |windows|.
 
+To <dfn>parse trigger specs</dfn> given a [=map=] |map|, a [=moment=]
+|sourceTime|, a [=duration=] |expiry|, a [=report window list=]
+|topLevelWindows|, and an unsigned 32-bit integer |defaultTriggerDataCardinality|:
+
+1. [=Assert=]: |defaultTriggerDataCardinality| is greater than 0 and less than
+    or equal to [=max distinct trigger data per source=].
+1. Let |specs| be a new [=map=].
+1. If |map|["`trigger_specs`"] does not [=map/exist=]:
+    1. Let |spec| be a new [=trigger spec=] with the following items:
+        : [=trigger spec/event-level report windows=]
+        :: |topLevelWindows|
+    1. For each integer |triggerData| between 0 (inclusive) and
+        |defaultTriggerDataCardinality| (exclusive):
+        1. [=map/Set=] |specs|[|triggerData|] to |spec|.
+    1. Return |specs|.
+1. If |map|["`trigger_specs`"] is not a [=list=] or [=list/is empty=], or its
```

Comment:
Is this the only case that setting an empty list is treated differently from not setting the field?

Reply:
Yes, but this field is different from other lists in that it enumerates valid configurations, so it is potentially misleading for the empty list to be treated as the default.

This is related to the current behavior of not allowing `null` for `debug_key`, for example.

Reply:
That makes sense, thanks for clarifying.